src/main/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionController.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.pip.subscription.management.controllers;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.pip.subscription.management.models.Subscription;
+import uk.gov.hmcts.reform.pip.subscription.management.models.SubscriptionDto;
+import uk.gov.hmcts.reform.pip.subscription.management.service.SubscriptionService;
+
+import java.util.UUID;
+
+
+@RestController
+@Api(tags = "Subscription management API")
+@RequestMapping("/subscription")
+public class SubscriptionController {
+
+    @Autowired
+    SubscriptionService subscriptionService;
+
+    @PostMapping(consumes = "application/json")
+    @ApiOperation("Endpoint to create a new unique subscription - the 'id' field is hidden from swagger as it is auto"
+        + " generated on creation")
+    @ApiResponses({
+        @ApiResponse(code = 200, message = "Returns a string with the subscription id and user id of the generated "
```

Comment:
if you could represent the literal response rather than a description of the response, matching how youve done it in the delete